docusaurus/docs/packages/pkg/relayer/relayminer_config.md

```diff
@@ -0,0 +1,379 @@
+```
+## `query_node_rpc_url`
+_`Required`_
+
+The RPC URL of the Pocket node that allows the `RelayMiner` to subscribe to events
+via websockets. It is then re-formatted as `ws://<host>/websocket` and establishes a
+persistent connection to the Pocket node to stream events such as latest blocks,
+application staking events, etc...
+
+## `query_node_grpc_url`
+_`Optional`_
+
+The gRPC URL of the Pocket node that allows the `RelayMiner` to query/pull data from
+the Pocket network (eg. Sessions, Accounts, etc...). If unspecified, `tx_node_grpc_url`
+value will be used.
+
+## `tx_node_grpc_url`
+_`Required`_
+
+The gRPC URL of the Pocket node that allows the `RelayMiner` to submit transactions.
+It may have a different host than the `query_node_rpc_url` or the `tx_node_grpc_url`
+but same values are acceptable too.
+
+# RelayMiner proxies
+
+The `proxies` section of the configuration file is a list of proxies that the
+`RelayMiner` will use to start servers by listening on the configured host.
+
+At least one proxy is required for the `RelayMiner` to start.
+
+```yaml
+proxies:
+  - name: <string>
+    type: <enum{http}>
+    host: <host>
+```
+
+## `name`
+_`Required`_, _`Unique`_
+
+Is the name of the proxy which will be used as a unique identifier to reference
+proxies in the [Suppliers](#suppliers) section of the configuration file.
+It corresponds to a server that will be started by the `RelayMiner` instance
+and must be unique across all proxies.
+
+## `type`
+_`Required`_
+
+The type of the proxy server to be started. Currently only `http` is supported.
+When other types are supported, the `type` field could determine if additional
+configuration options are allowed be them optional or required.
+
+## `host`
+_`Required`_, _`Unique`_
+
+The host to which the proxy server will be started and listening to. It must be
+a valid host according to the `type` filed and must be unique across all proxies.
+
+# Suppliers
+
+The `suppliers` section of the configuration file is a list of suppliers that
+represent the services that the `RelayMiner` will offer to the Pocket network
+through the configured proxies and their corresponding services to where the
+requests will be forwarded to.
+
+At least one supplier is required for the `RelayMiner` to be functional.
+
+```yaml
+suppliers:
+  - name: <string>
+    type: <enum{http}>
+    service_config:
+      url: <url>
+      authentication:
+        username: <string>
+        password: <string>
+      headers:
+        <key>: <value>
+    hosts:
+      - <host>
+    proxy_names:
+      - <string>
+```
+
+## `name`
+_`Required`_, _`Unique`_
+
+The name of the service which will be used as a unique identifier to reference
+a service provided by the `Supplier` and served by the `RelayMiner` instance.
+
+## `type`
+_`Required`_
+
+The transport type that the service will be offered on. It must match the `type` field
+of the proxy that the supplier is referencing through `proxy_names`. Currently only
```

Comment:
Try to avoid using `Currently` in documentation.

It will 100% become techdebt & legacy very quickly.